koberlander/Music-Icons-Concert
# Reorganize files to get basic web app loading

# How to accept and test these changes

- Merge this pull request on github web site
- In your local code repo on the Music-Icons-2 branch, do a `git pull` to get the latest code
- `cd server && npm install && cd -` to install the node dependencies
- From the repo root directory start the server with `node server`. It should have your app on `http://localhost:3000`
- All the URLs now load properly but the angular app itself still needs a few issues debugged.

# What I Changed to get to this point

- Separate server code into server directory
- Put client code into wwwroot directory
- Get basic static assets node/express server working
- Fix numerous syntax errors in javascript
- Fix some of angular app initialization
- App still does not load entirely and socket.io code should probably start over
